users/templates/users/password_reset_form.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+{% load crispy_forms_tags %}
+
+{% block content%}
+    <div class="container text-center mt-2 container-reset">
+        <h1>Password Reset Form</h1>
+        <p style="width: 60%; margin: 0 auto">
+            Please enter your new password so we can verify.
+        </p>
+        <br />
+        <form action="" method="POST">
+            {% csrf_token %}
+            {% if form.errors %}
```

Comment:
This is not what I asked for. I asked for NON_FIELD_ERRORS. Again, check registration.